package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "api.kitchen.support",
+  "version": "0.0.1",
+  "description": "Kitchen Support's RESTful API backend.",
+  "author": "Garrick Buckley <gbuckle@purdue.edu>",
+  "contributors": [
+    {
+      "name": "Garrick Buckley",
+      "email": "gbuckle@purdue.edu"
+    },
+    {
+      "name": "Jack Horton",
+      "email": "horton7@purdue.edu"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kitchensupport/api.git"
+  },
+  "dependencies": {
+    "express": "*",
+    "browserify": "*",
+    "gulp-ruby-sass": "*",
```

Comment:
Also, we don't really have a production "branch" or a dev "branch". this is our branch for both, and all of the gulp and browserify tools are most definitely development tools. This is a really esoteric thing because it doesn't overly matter, but it is more semantically correct if we have our dev tools in devDependencies.

I would argue, though, that we should be using versions that we know are compatible with our codebase, more so than any version. I dont want to run `npm update` and have something break.